src/pages/billing/List.js

```diff
@@ -145,160 +179,112 @@ const NewBillingProjectModal = ajaxCaller(class NewBillingProjectModal extends C
           ])
         ])
       ]),
-      submitting && spinnerOverlay
+      isBusy && spinnerOverlay
```

Comment:
Do you mean that the spinner should be limited to the modal, rather than being a `spinnerOverlay`? AFAIK if the user clicks out of the modal while it is submitting, it will cancel the creation so I think we do want to block the rest of the app to discourage people from clicking outside of it.

Reply:
There's a separate prop you can pass to the modal to prevent it from being closed that way, take a look at my account creation modal.